docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -75,60 +91,22 @@ Preparing For a Release
 
    The version number must adhere to :ref:`the Flocker version numbering policy <version-numbers>`.
 
-#. Export the version number of the release being created as an environment variable for later use:
-
-   .. prompt:: bash $
-
-      export VERSION=0.1.2
-
 #. Create an issue in JIRA:
 
-   This should be an "Improvement" in the current sprint, with "Release Flocker $VERSION" as the title, and it should be assigned to yourself.
+   This should be an "Improvement" in the current sprint, with "Release Flocker <version-being-released>" as the title, and it should be assigned to yourself.
    The issue does not need a design, so move the issue to the "Coding" state.
 
-#. Create and log in to a new :doc:`Flocker development machine <vagrant>`:
+#. Create an environment to do a release in:
 
-   This uses SSH agent forwarding so that you can push changes to GitHub using the keys from your workstation.
+   .. prompt:: bash $,(flocker-release)tmp-1234$,(flocker-release)flocker$ auto
 
-   Add your SSH key to the ``sshd`` agent.
-   Note that the ssh key you use must be linked to your GitHub account.
-
-   .. prompt:: bash $
-
-      [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent)
-      ssh-add $HOME/.ssh/id_rsa
-
-   This copies your local git configuration from ``~/.gitconfig``.
-   If this does not exist, commits made for the release will be associated with the default Vagrant username and email address.
-
-   This copies your local configuration for `S3`_ from ``~/.aws``.
-   If this does not exist, a later step will create it.
-
-   .. prompt:: bash $
-
-      git clone git@github.com:ClusterHQ/flocker.git "flocker-${VERSION}"
-      cd flocker-${VERSION}
-      vagrant up
-      vagrant scp default:/home/vagrant/.bashrc vagrant_bashrc
-      echo export VERSION=${VERSION} >> vagrant_bashrc
-      vagrant scp vagrant_bashrc /home/vagrant/.bashrc
-      if [ -d ~/.aws ]; then vagrant scp "~/.aws" /home/vagrant; fi
-      vagrant ssh -- -A
-
-#. Create a release branch, and create and activate a virtual environment:
-
-   .. prompt:: bash [vagrant@localhost]$
-
-      # The following command means that you will not be asked whether
-      # you want to continue connecting
-      ssh-keyscan github.com >> ~/.ssh/known_hosts
-      git clone git@github.com:ClusterHQ/flocker.git
-      cd flocker
-      mkvirtualenv flocker-release
-      pip install --editable .[dev]
-      admin/create-release-branch --flocker-version="${VERSION}"
-      admin/update-license
-      git commit -am "Updated copyright in LICENSE file"
+      $ mktmpenv --prompt="(flocker-release)"
```

Comment:
The use of `mktempenv` makes it difficult to deal with the gap during tests.  I know you want to remove `$VERSION` from the second part of the release, to avoid the user having to keep it set.  However, in my case, I needed to reboot for updates, and then had to find the name of the temporary virtualenv to reconnect.  Luckily, I only had one tempenv to choose from. But having to remember the tempenv name is as bad as having to ensure `VERSION` is always set.  I suggest just going to a named virtualenv, probably `flocker-${VERSION}` since `VERSION` needs to be set for this stage of the release.